### PR TITLE
Cleaned up links

### PR DESCRIPTION
**Pull request by StingRayZA**
Changed intra repo links to point to other files in the repo, rather to files in a specific repo.
Fixed a couple of other hard links to the legacy (paypal) repo.
Updated one reference of innersourcecommons.com to innersourcecommons.org for consistency.

### CODE REVIEW

File: project-roles/trusted-committer.md [hunk 1]
```diff
@@ -163,5 +163,5 @@ Published internally at Nike; drafted via pull-request in June of 2018.
 [Loren Sanz]: https://github.com/mrsanz
 [Jeremy Hicks]: https://github.com/greatestusername
 [Noah Cawley]: https://github.com/utanapishtim
-[praise]: https://github.com/paypal/InnerSourcePatterns/blob/master/praise-participants.md
+[praise]: https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/praise-participants.md
```
Comment (StingRayZA) [inline project-roles/trusted-committer.md]: I'm not sure how to make that a relative link...

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-This repository serves as the collection-point, ideation hub, and process behind the [InnerSource Commons'](http://innersourcecommons.com) InnerSource Patterns--a set of proven and reviewed solutions to InnerSource problems. These patterns illustrate beneficial activities and behaviors found in [organizations who apply InnerSource methodologies](http://paypal.github.io/InnerSourceCommons/getting-started/). See below the list for more on what a pattern is, how to use them in your organization, and how to get involved.
+This repository serves as the collection-point, ideation hub, and process behind the [InnerSource Commons'](http://innersourcecommons.org) InnerSource Patterns--a set of proven and reviewed solutions to InnerSource problems. These patterns illustrate beneficial activities and behaviors found in [organizations who apply InnerSource methodologies](http://paypal.github.io/InnerSourceCommons/getting-started/). See below the list for more on what a pattern is, how to use them in your organization, and how to get involved.
```
Comment (lenucksi) [inline README.md]: Innersourcecommons.org currently does a redirect to the Paypal repo, could you change the ```organizations who apply InnerSource methodologies``` link to use the redirect there? 

We might change things around there later, such that a second round of search&replace might be warranted but that would then just remove a part of the URL not change it entirely.

**Review by lenucksi — APPROVED** [review]
This looks good to me. I don't know about the relative link part comment either, maybe someone else, e.g. @rrrutledge or @maxcapraro knows. 
One minor nit, let's find someone for the open question and then merge it I say.

File: README.md (hunk 2 above)
Comment (StingRayZA) [reply README.md]: 👍 good catch. updated 😁 
I couldn't find any other references to paypal.github.io...

File: project-roles/trusted-committer.md (hunk 1 above)
Comment (StingRayZA) [reply project-roles/trusted-committer.md]: figured it out thanks to [github docs](https://github.github.com/gfm/#reference-link)
(TIL)

**Review by rrrutledge — APPROVED** [review]
Great initiative, Raymond!  Good call on using relative links.

**Comment by lenucksi** [comment]
Thanks for cleaning up the links and making them relative!

**Pull request merged**